src/main/kotlin/io/Ust.kt

```diff
@@ -192,17 +219,46 @@ object Ust {
             line.tryGetValue("VBR")?.let {
                 pendingVBR = it.split(',').mapNotNull { cell -> cell.toDoubleOrNull() }
             }
+            line.tryGetValue("Piches")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+            line.tryGetValue("Pitches")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+            line.tryGetValue("PitchBend")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+        }
+        val pitchDataMode1 = UtauMode1TrackPitchData(notePitchDataListMode1)
+        val pitchDataMode2 = notePitchDataListMode2.ifEmpty { null }?.let { UtauMode2TrackPitchData(it) }
+        return FileParseResult(file, projectName, notes, tempos, isMode2, pitchDataMode1, pitchDataMode2)
```

Comment:
I think we should only import pitch from mode1 OR mode2, to improve performance.
The problem is I could not change the `isMode2` in UST to `false` even if I selected mode1 in UTAU. any information?

Reply:
I agree with the performance concern. I will change the behaviour later.
As for "Mode2=true" field in UST, UTAU has a strange behaviour. It only indicates an UST as a mode2 UST when there is "Mode2=true" in the header section (And it's case sensitive). Furthermore, UTAU will not save this field when it saves a mode1 ust (In other words, there will be no texts like "Mode2=false" in UST.

Reply:
Ok, I see